distributed/client.py

```diff
@@ -581,6 +594,8 @@ class Client:
 
     _instances = weakref.WeakSet()
 
+    default_event_handlers = {"print": _handle_print, "warn": _handle_warn}
```

Comment:
Nitpick: Is this something we want users to have interact with? If not, I have a slight preference for prepending an underscore here (i.e. `_default_event_handlers`)